### PR TITLE
[IMP] web : Tutorial chgo

### PR DESCRIPTION
**Pull request by chgo-odoo**
This PR follows the tutorial of the R&D UX/UI

Requires
- https://github.com/odoo/enterprise/pull/38340

task-3234820

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jpp-odoo** [comment]
@chgr2 
Hello, you forgot this part :
> 6 Push both branches and open PRs (draft mode) -> Ensure that Runbot in grenn -> Process reviewer's feedback 

If you create a normal PR and not a draft one, the corresponding teams are automatically ping to review.

I change your PR to draft, and remove the reviewers.

**Review by Brieuc-brd — CHANGES_REQUESTED** [review]
Hello @chgr2, thank you for your work 💪 

I have left some comments on what needs to be fixed.

Also, I think there is a misunderstanding on points 1 and 2. 😅 
You have applied different colors for the **primary button** in Community and Enterprise and I don't see any change on the **secondary button**. 
You should re-read the exercise 😉 

Regarding the commit titles :
- Keep the caps only for module names
- Try not to exceed 50 characters

For example, you could change 
`[IMP] web : Changed 'o_burger_menu' class to 'o_hamburger_menu'`
 to `[IMP] web : adapt 'o_burger_menu' class` and add details in the commit description.

👉 https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html


Feel free to ask me if you have any questions 👍.

File: addons/web/static/src/scss/primary_variables.scss [hunk 1]
```diff
@@ -186,3 +186,6 @@ $o-btn-secondary-hover-border: $o-gray-200 !default;
 $o-btn-light-bg: $o-white !default;
 $o-btn-light-border: $o-btn-light-bg !default;
 $o-btn-light-background-hover: $o-gray-100 !default;
+
+// Borders
+$hr-color: pink !default;
```
Comment (Brieuc-brd) [inline addons/web/static/src/scss/primary_variables.scss:191]: Good job 👍 
But the exercise states that the custom color should only be applied in Community.
For now it applies to both.

**Review by Brieuc-brd — CHANGES_REQUESTED** [review]
Thanks for the update @chgr2 ! It’s good, but it’s still perfectible ;)

I let you some comments on both PR.

About the commits, I see two issues :
- I think the title and the description of 5199d8e89988a82fa0f6d5a14c16470360c8adea are not correct.
It describes a “reset of border color” but only the `$o-community-color` is changed.

- For 276b4d5b65470dcafb94faf01ea36d569d4690eb and e38ebbe96571b0bc65888f5de4dba46f864bb64a you can shorten the title
(ideally < 50 chars : https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html)

Courage 💪

**Comment by Brieuc-brd** [comment]
LGTM, good job 👍 

There is still an error on runbot regarding the gantt view, but I don't think it's related to your changes, what do you think @stefanorigano ?

**Pull request closed without merging**